spark-3.4/clickhouse-spark/src/main/scala/xenon/clickhouse/func/Util.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package xenon.clickhouse.func
+
+object Util {
+  def intHash64Impl(x: Long): Long =
+    // https://github.com/ClickHouse/ClickHouse/blob/f4c73e94d21c6de0b1af7da3c42c2db6bf97fc73/src/Functions/FunctionsHashing.h#L143
```

Comment:
better to use a release tag instead of a commit hash

Reply:
Pushed a new commit fixing this